scripts/benchmark/contract_data/ContractFactory.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.4.19;
+
+contract ContractFactory{
```

Comment:
This looks like it may have snuck in un-intentionally?

Reply:
No, this is intentional, this is the contract that will be used for testing the DOS benchmarks, however it is currently poorly named

Reply:
Ah, could we keep that separate and focus on just the ERC20 benchmarks in this PR and do the DOS benchmarks in a new PR?